api/src/main/java/org/openmrs/scheduler/Schedule.java

```diff
@@ -9,13 +9,16 @@
  */
 package org.openmrs.scheduler;
 
+import org.hibernate.envers.Audited;
```

Comment:
Are we using this anywhere?

Reply:
We are auditing Schedule entity

Reply:
Where in the codebase are we using that entity?

Reply:
I added auditing to it because it is used here: https://github.com/openmrs/openmrs-core/blob/3a7925a3484c098078db6e70f9b6f4a6cbb4e884/api/src/main/resources/org/openmrs/scheduler/db/hibernate/Schedule.hbm.xml#L16

However, I couldn't find any scheduler_schedule tables in the database.

Reply:
Looks like it may be time to get rid of it. 😊

Reply:
Yes, but in a separate pull request.

Reply:
I created a PR for this: https://github.com/openmrs/openmrs-core/pull/4670